canonical/ubuntu.com
# Refresh static contact form for /kernel/real-time

## Done

- Update design for static form on `/kernel/real-time/contact-us`
- Design: https://www.figma.com/design/mxftgCWDJ6s3LrlaedREkP/24.10-Static-forms?node-id=1-3&t=wJYqljxJE6kXFa8h-0
- Make form labels more accessible
- Fix missing payload on marketo submission

## QA
- Go to [demo](https://ubuntu-com-14044.demos.haus/kernel/real-time/contact-us)
- Check that the demo adheres to the design
- Check that required fields work as expected
- Once submitted, check that payload is correct (included screenshots below)
- Check that form payload is submitted to marketo successfully

## Issue / Card

Fixes [WD-13103](https://warthogs.atlassian.net/browse/WD-13103)

## Screenshots
![Screenshot 2024-07-09 at 3 33 55 PM](https://github.com/canonical/ubuntu.com/assets/62298176/ebba7bf4-01d2-480f-8ebe-b1eb4ee025c3)
![Screenshot 2024-07-09 at 3 34 03 PM](https://github.com/canonical/ubuntu.com/assets/62298176/2bbf6f62-0c51-42bf-8798-d5ee42f49270)



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-12998]: https://warthogs.atlassian.net/browse/WD-12998?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WD-13103]: https://warthogs.atlassian.net/browse/WD-13103?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
